src/templates/index.hbs

```diff
@@ -367,25 +383,25 @@
               <br>
 
 
-          <div class="row">
-             <div class="col-md-offset-1 col-md-3">
-               <img src="/assets/img/andreluisgomes.jpeg" class="img-responsive img-circle" />
+          <div class="row staff">
+             <div class="col-sm-4">
+               <img src="/assets/img/andreluisgomes.jpeg" class="img-circle" />
                <h5>André Luis Gomes @Invillia</h5>
                    <span class="input-group-btn">
                     <button class="btn btn-default btn-green" type="button"> <a href="https://www.linkedin.com/in/andreluisgomes/" target="_blank"> Who? </a></button>
                   </span>
                 </div>
 
-                <div class="col-md-3">
-                  <img src="/assets/img/josedamiani.jpg" class="img-circle img-responsive" alt="" />
+                <div class="col-sm-4">
```

Comment:
`<div class="col-sm-4 col-xs-4"> ` 
to xs and small devices....

Reply:
I don't think this is correct. You don't need to use multiple col-* classes with the same number of columns, `col-xs-4` would suffice in the case you proposed. Nonetheless in this scenario I want the columns to occupy an entire row on `xs` screens, that is why there is only a `col-sm-4` there (i.e. `col-xs-12` is the effective rule being applied for extra small screens).

Reply:
In the case before I only used col-md for not worrying about the mobile part for this site, but I knew that we had a problem when rendering the team on the mobile because the images of the members were very large. So I recommend col-xs and col-md membership ... in the image below you can see the exact problem of the generic breakpoint.

![image](https://user-images.githubusercontent.com/4699770/49540276-cbb25b00-f8b6-11e8-89a2-290591b0739e.png)

Reply:
But this problem is fixed now (the team is rendering correctly on extra-small resolutions, one member per row with their picture nicely sized).

IMO there is no room for 3 members in a single row for extra-small screens (so we should not add `col-xs-4`, it should start at the `sm` level).

Reply:
For reference:

Large resolution:

![captura de tela_selecionar area_20181205193626](https://user-images.githubusercontent.com/352108/49545549-23f05980-f8c5-11e8-81f6-a553ee5de9d3.png)

Extra-small resolution:

![captura de tela_selecionar area_20181205193644](https://user-images.githubusercontent.com/352108/49545554-294da400-f8c5-11e8-9f70-62837bc1e1ee.png)

Reply:
🚀